src/rgw/rgw_cr_rados.cc

```diff
@@ -579,7 +579,7 @@ int RGWAsyncStatRemoteObj::_send_request()
                        nullptr); /* string *petag, */
 
   if (r < 0) {
-    ldout(store->ctx(), 0) << "store->fetch_remote_obj() returned r=" << r << dendl;
+    ldout(store->ctx(), 0) << "store->stat_remote_obj() returned r=" << r << dendl;
```

Comment:
@ZVampirEM77 I am sorry. Still in recent master function before this statement is `store->fetch_remote_obj` and in your PR also you have not changed from `store->fetch_remote_obj` to `store->stat_remote_obj`.

Reply:
Yeah. Thanks for all your suggestion. I will fix those issues later.

In this comment, you mean I should create a new PR to fix the `store->fetch_remote_obj` typos?